modules/create_timepoint/php/endpoints/timepoint.class.inc

```diff
@@ -0,0 +1,339 @@
+<?php declare(strict_types=1);
```

Comment:
Is it necessary to add sub-folder endpoints under php? If you insist on using this, you need to modify test/run-php-linter.sh to pass Travis. (add the endpoints into line 6 to make the checkstatic works)

 

Reply:
> you need to modify test/run-php-linter.sh to pass Travis

Why does this make Travis fail? The command looks like it should check every PHP file under `modules/`

Reply:
see: https://github.com/aces/Loris/pull/5249

Reply:
@xlecours should that be merged into major and not minor?